sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -20,14 +16,21 @@ npm install -g autorest
 There is one swagger for Chat management APIs. 
 
 ```ps
-cd <swagger-folder>
-autorest README.md --java --v4 --use=@autorest/java@4.0.2
+> cd <swagger-folder>
+> autorest README.md --java  --android --v4 --use=@autorest/java@4.0.26
 ```
 
 ## Update generated files for chat service
 To update generated files for chat service, run the following command
 
-> autorest README.md --java --v4 --use=@autorest/java@4.0.11
+> autorest README.md --java --android --v4 --use=@autorest/java@4.0.26
+
+Rename generated ChatsImpl class to ChatImpl;
+Rename generated ChatThreadsImpl to ChatThreadImpl;
+Modify AzureCommunicationChatServiceImpl
+    Modify method getChats name to getChatClient
+    Modify method getChatThreads name to getChatThreadClient
+Restore file chat/models/ChatMessageType.java
```

Comment:
Maybe we should consider keeping files with code-gen produced "names" like in java (instead of renaming files post generation)